corehq/apps/api/resources/v0_5.py

```diff
@@ -312,6 +312,30 @@ def _update(cls, bundle, user_change_logger=None):
 
                         if change_messages:
                             user_change_logger.add_change_message({'phone_numbers': change_messages})
+                elif key == 'default_phone_number':
+                    old_phone_numbers = set(bundle.obj.phone_numbers)
+                    phone_number = bundle.data.get('default_phone_number')
+                    if not isinstance(phone_number, str):
+                        errors.append(_('Only a single value, not a list, can be set for default_phone_number.'))
```

Comment:
I imagine this could also be an integer is the API request wasn't properly formatted. Maybe update the message to say it must be a string.

Reply:
Good idea. See f53f0802cec512b73cde70558feed4a73acc663d